packages/core/parcel/src/cli.js

```diff
@@ -135,7 +135,7 @@ applyOptions(watch, commonOptions);
 let build = program
   .command('build [input...]')
   .description('bundles for production')
-  .option('--no-minify', 'disable minification')
+  .option('--no-optimize', 'disable optimizers')
```

Comment:
Well, it doesn't do that yet

Reply:
Yeah because we don't have an alternative yet for the postprocessing that isn't actually an optimisation...

Should I change the flag and not the description?

Was just thinking to get this in and introduce the new plugin in a seperate PR

Reply:
> Should I change the flag and not the description?

👍